warehouse/templates/includes/current-user-indicator.html

```diff
@@ -29,6 +29,10 @@
           Admin
         </a>
         {% endif %}
+        <a class="dropdown__link" href="{{ request.route_path('accounts.profile', username=request.user.username) }}">
+          <i class="fa fa-cube" aria-hidden="true"></i>
```

Comment:
This should probably have a different icon than the "Your Projects" link.

Reply:
Good point, thanks. Do you have any suggestions?

Reply:
We use [`user-circle`](https://fontawesome.com/icons/user-circle?style=solid) for this elsewhere, that might be the best choice but I'll leave it to @nlhkabu.

Reply:
+1 for `user-circle`